config/backends.go

```diff
@@ -170,3 +175,36 @@ func (cfg *Redis) validateAndLog() error {
 	log.Infof("config.backend.redis.tls.insecure_skip_verify: %t", cfg.TLS.InsecureSkipVerify)
 	return nil
 }
+
+type Ignite struct {
+	Scheme  string            `mapstructure:"scheme"`
+	Host    string            `mapstructure:"host"`
+	Port    int               `mapstructure:"port"`
+	Secure  bool              `mapstructure:"secure"`
```

Comment:
I think this should be renamed `VerifyCert` or some documentation added to explain that secure in this context means verifying the SSL certificate on the ignite server. On the face of it `Secure` sounds like it may be redundant with `Scheme`, or suggest that there is a third possibility for `Scheme` that may or may not use a secure connection.

Reply:
Modified and added a comment